libcontainer/container_linux.go

```diff
@@ -506,6 +520,48 @@ func isDmzBinarySafe(c *configs.Config) bool {
 	return false
 }
 
+// fixStdioPermissions fixes the permissions of STDIO to the specified user.
+func fixStdioPermissions(uid int) error {
+	var null unix.Stat_t
```

Comment:
Can you please move it back to `init_linux.go`? This will make the patch shorter and review simpler.

Reply:
Since this function is needed in both parent and init process, so I think it should be in `container_linux.go`.
And I have updated the PR, so it really should be moved to `container_linux.go`.